src/+otp/+linear/LinearProblem.m

```diff
@@ -40,5 +40,19 @@ function onSettingsChanged(obj)
             obj.RHSPartitions = cellfun(@obj.createRHS, obj.Parameters.A, ...
                 'UniformOutput', false);
         end
+        
+        function y = internalSolveExactly(obj, t)
+            numT = length(t)
+            for i = 1:numT
+                yi = expm(t(i) * obj.RHS.Jacobian) * obj.Y0;
```

Comment:
Shouldn't this technically be dependent on t(1) or on obj.TimeSpan(1)? I would do t(i) - obj.TimeSpan(1)